apps/001-hello/app.R

```diff
@@ -28,12 +28,40 @@ ui <- fluidPage(
       plotOutput(outputId = "distPlot")
 
     )
-  )
+  ),
+
+  # include shinyjster JS at end of UI definition
+  shinyjster_js("
```

Comment:
Needs to namespace the function since it was not `library`ed. 

```r
shinyjster::shinyjster_js("
```

Reply:
I have added that change Barret.